tests/drivers/adc/adc_api/boards/stm32l4r9i_disco.overlay

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Please split this in a dedicated commit.

Reply:
Done.